### DISCUSSION TITLE
Notify push self test failed

### DISCUSSION THREAD

**Discussion by addy419** [nextcloud/all-in-one#3837] [Questions] [answered]
Hi, in the logs for the notify push container, I am getting an error
```
ERROR [notify_push] src/main.rs:78: Self test failed: Error while communicating with nextcloud instance
```
The container is green and everything works fine (mostly) but there are no other logs, just that. How do I solve this issue?

Nextcloud AIO v7.7.0 on Ubuntu 22.04

**Comment by szaimen** [comment, marked as answer]
Hi, as long as there is no further error logged, everything works correctly. But it will still be improved with https://github.com/nextcloud/all-in-one/pull/3843

**Reply by szaimen** [reply]
This is now released with v7.8.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Reply by addy419** [reply]
@szaimen I'm on v7.10.0 and still received [2024-01-18 15:27:32.291414 +00:00] ERROR [notify_push] src/main.rs:78: Self test failed: Error while communicating with nextcloud instance

**Reply by szaimen** [reply]
As long as it is only logged once, everything is working fine.

**Reply by Itchyfantastico** [reply]
Same error here and the status of Notify Push remains yellow "Starting"

**Reply by szaimen** [reply]
Hi @Itchyfantastico, see https://github.com/nextcloud/all-in-one/issues/4092#issuecomment-1903623951
